patches/chrome-browser-safe_browsing-download_protection-check_client_download_request_base.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+index 4f7fd86c5e6edd387254d0eab6aa399db5e42105..1edf51b8e80c967e275e327d8143973ec4881871 100644
+--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
++++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+@@ -554,6 +554,7 @@ void CheckClientDownloadRequestBase::SendRequest() {
+   request->set_archive_directory_count(directory_count_);
+   request->set_request_ap_verdicts(is_under_advanced_protection_);
+ 
++  BRAVE_SEND_REQUEST_FILTER
```

Comment:
you can avoid this patch by subclassing CheckClientDownloadRequest and doing a chromium_src override in DownloadProtectionService

Reply:
Unfortunately that doesn't work because `PopulateRequest()` runs [before](https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc;l=505;drc=73ea864deabdd9daeca8f82228653ef2756ef6b8) some of the data is [added](https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc;l=513-531;drc=73ea864deabdd9daeca8f82228653ef2756ef6b8) to the `request` object.